tests/foreman/cli/test_capsule.py

```diff
@@ -163,3 +163,30 @@ def test_ssl_capsule(self):
         @Status: Manual
 
         """
+
+    def test_enable_bmc(self):
+        """@Test: Enable BMC feature on smart-proxy
+
+        @Feature: Capsules
+
+        @Setup: A capsule installed with SSL enabled.
+
+        @Steps:
+
+        1. Enable BMC feature on proxy by running installer with:
+           katello-installer --foreman-proxy-bmc 'true'
```

Comment:
You can denote inline code blocks with double backticks, like so:

    1. Enable BMC feature on proxy by running installer with:
        ``katello-installer --foreman-proxy-bmc 'true'``

Comment is not a blocker.